operations/observability/mixins/cross-teams/dashboards/gitpod-overview.json

```diff
@@ -96,7 +96,8 @@
                 "steps": []
               },
               "unit": "none"
-            }
+            },
+            "overrides": []
```

Comment:
Unfortunately duplicating the Overview dashboard and downloading its JSON also added a number of unrelated changes to the diff, I think these are changes from newer Grafana versions that haven't been persisted yet to git?

I've only really updated the "Running workspaces" panels.